libs/hwManager/manufacturers/ledger/index.js

```diff
@@ -70,7 +67,6 @@ const listener = (transport, actions) => {
  */
 const getLedgerAccount = (index = 0) => {
   const ledgerAccount = new LedgerAccount();
-  ledgerAccount.coinIndex(SupportedCoin.LISK);
```

Comment:
In the past the library was expecting coin index to be explicitly specified as the same library was used by ark and other lisk compatible protocols, so with new lib its no longer the case, by default it will use lisk coin index.